docs/rook.md

```diff
@@ -1,17 +1,21 @@
 # Rook
 
-Rook is parent command which requires sub-command to get working. Currently, rook supports these sub-commands:
+The `rook` command supports the following sub-commands:
 
-1. `purge-osd <osd-id> [--force]` : [purge osd](#purge-osd) permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.
+1. `purge-osd <osd-id> [--force]` : [purge osd](#purge-osd) permanently remove an OSD from the cluster. Multiple OSDs can be removed in a single command with a comma-separated list of IDs.
 2. `version`: [version](#version) prints the rook version.
 3. `status`      : [status](#status) print the phase and conditions of the CephCluster CR
 4. `status all`  : [status all](#status-all) print the phase and conditions of all CRs
 5. `status <CR>` : [status  cr](#status-cr-name) print the phase and conditions of CRs of a specific type, such as 'cephobjectstore', 'cephfilesystem', etc
 
-## Purge OSD
+## Purge an OSD
 
-Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.
+Permanently remove an OSD from the cluster. 
 
+
+!!! warning
+    Data loss is possible when passing the --force flag if the PGs are not healthy on other OSDs.
```

Comment:
I'm not sure that all users will know what a PG is. It might be good to link to ceph docs.

Reply:
good point, @BlaineEXE ! would you consider it sufficient to add that in a follow-up PR?

Reply:
My experience is that most follow-up PRs don't actually happen. This is already merged, so it'll have to be in follow-up